src/app/shared/components/template/components/button/button.component.scss

```diff
@@ -67,6 +68,52 @@ ion-button[data-param-style~="card"] {
   }
 }
 
+.button-container[data-variant~="card-portrait"] {
+  background-color: white;
+  color: var(--ion-color-primary);
+  font-size: var(--buttons-font-size-small);
+  font-weight: var(--font-weight-bold);
+  border: 1px solid var(--ion-color-gray-light);
+  border-radius: var(--ion-border-radius-secondary);
+  min-height: var(--buttons-short-height);
+  filter: drop-shadow(var(--ion-default-box-shadow));
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1.5rem 0.5rem 0.5rem;
+  // TODO: make a variable, shared with task card
+  width: 165px;
+
+  &:active {
+    background-color: var(--ion-color-gray-light);
+  }
+
+  img {
+    width: 85%;
+  }
+
+  .button-text {
+    width: 100%;
+    padding: 1rem 1rem 0.2rem;
```

Comment:
nit(blocking) 0.2rem seems pretty arbitrary, better to avoid fine-tuning so much on one component and use numbers more common throughout (for consistency across components). Also given 1rem is 16px on most displays 0.2rem would be 3.2px which doesn't make much sense